llvm/test/CodeGen/AMDGPU/no-hsa-graphics-shaders.ll

```diff
@@ -1,19 +0,0 @@
-; RUN: not llc -mtriple=amdgcn-unknown-amdhsa < %s 2>&1 | FileCheck %s
-
-; CHECK: in function pixel_s{{.*}}: unsupported non-compute shaders with HSA
```

Comment:
We must maintain test coverage for error cases

Reply:
I wanted to adapt the test, but FileCheck didn't seem to be able to check for the `LLVM ERROR ...` line (first line of output/stderr). So I assumed that it may not be possible to check for fatal errors. But now the situation is different anyway :)

Reply:
Yeah, in the meantime, I found this by searching through similar tests (basically `grep -rl "LLVM ERROR"` ^^). However, it still somehow skipped the first line, even though stderr was redirected to stdout. But this is no longer an issue since I removed the fatal error again, based on the feedback I received here.